aptos-move/aptos-aggregator/src/delayed_field_extension.rs

```diff
@@ -637,7 +637,7 @@ impl AggregatorData {
         BTreeSet<AggregatorVersionedID>,
```

Comment:
Does AggregatorerVersionedID have to be renamed as we renamed AggregatorID?

Reply:
no, that one is only used for AggregatorV1 vs AggregatorV2 in the extension, as far as I can see. 

we always use DelayedFieldID for Snapshots